runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePlugin.java

```diff
@@ -57,6 +57,11 @@ MetronomePluginConfiguration provideConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(MetronomePluginConfiguration.class);
 	}
+	@Override
```

Comment:
Separate these methods with a blank line
```suggestion

	@Override
```